src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -286,8 +280,7 @@ export function getStatusContents(statusType, details = {}, name = {}) {
     case STATUS_TYPES.atVso:
       contents.title = 'Your appeal is currently with your Veteran Service Organization';
```

Comment:
Please make these edits:
- Delete "currently" (it's unnecessary/unspoken)
- Add "s" to the end of "Veteran" for "Veterans Service Organization"